app/views/submissions/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2><%=t('submissions.actions.new')%></h2>
```

Comment:
There's a neat shortcut that Rails provides if you name the locale keys after the view path. In this case we have `submissions/new.html.erb`, so if you do this in `en.yml`:

```yml
submissions:
  new:
    title: Create a new submission
```

the view can then use it like `<%= t(".title") %>` (the leading `.` is important)